pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -52,6 +52,7 @@ let
     , extraPoliciesFiles ? []
     , libName ? browser.libName or "firefox" # Important for tor package or the like
     , nixExtensions ? null
+    , hasMozSystemDirPatch ? pname != "firefox-bin"
```

Comment:
`firefox-beta-bin` & `firefox-devedition-bin` are also worth considering, but beyond that, `wrapFirefox` is used by `firefox-mobile`, `librewolf`, `floorp`, `midori`, `vimb`, `thunderbird`, `thunderbird-bin`, and `betterbird`. It might be better to check whether `browser.patches` exists and includes the relevant patch.

Reply:
Sigh, that gets more complicated that I want it to be. How about we check if the pname ends in `-bin`?

Reply:
ammended that but with prefix/suffix instead of a regex.